python/test/dynamo/importer_dynamic_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2023 Nod Labs, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import logging
+import unittest
+
+import torch
+import torch._dynamo as dynamo
+from torch._export import dynamic_dim
+# from torch._export.constraints import constrain_as_size, constrain_as_value
+from shark_turbine.dynamo.importer import FxImporter
+from shark_turbine.dynamo.passes import turbine_cpu_pass_pipeline
+import torch
+import torch._dynamo as dynamo
+from torch._dynamo.backends.common import aot_autograd
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch._decomp import get_decompositions
+from torch.func import functionalize
+from torch.fx import (
+    GraphModule,
+)
+
+def import_compiler(gm: GraphModule, example_inputs, decompose_ops=None):
+    imp = FxImporter()
+    if decompose_ops is not None:
+        gm = make_fx(
+            functionalize(gm),
+            decomposition_table=get_decompositions(decompose_ops),
+        )(*example_inputs)
+
+    gm.print_readable()
+    try:
+        imp.import_graph_module(gm)
+    finally:
+        print(imp.module)
+    imp.module.operation.verify()
+    return gm
+
+class DynamicBMM(torch.nn.Module):
+    def __init__(self, n, k):
+        super().__init__()
+        self.weight0 = torch.nn.Parameter(torch.rand(n, k))
+
+    def forward(self, inp, *, bias):
+        mm = torch.matmul(inp, self.weight0)
+        biased = mm + bias
+        return {"result": biased}
+
+class DynamicBuiltinOps(torch.nn.Module):
+    def forward(self, inp):
+        x = inp.size()[1] - inp.size()[2]
+        x = x * inp.size()[1] - 34.2
+        g = x / 32
+        return {"result": g}
+
+class ProgramTests(unittest.TestCase):
```

Comment:
Can we rename this to ImportSmokeTests? We aren't really testing anything with these other than that the importer doesn't crash or produce invalid IR (? - I think). Would be good to beef these up to real integration tests at some point.